src/main/kotlin/com/bardsoftware/backend/fleet/rmarkdown/PandocCompiler.kt

```diff
@@ -0,0 +1,31 @@
+package com.bardsoftware.backend.fleet.rmarkdown
+
+import org.slf4j.LoggerFactory
+import java.nio.file.Path
+
+val PANDOC_DEFAULT_VALUE = "pandoc"
+
+private val LOGGER = LoggerFactory.getLogger("Pandoc")
+val PANDOC_DEFAULT_FONT = "DejaVu Sans"
+
+fun compile(pandocCompileCommand: String, mainFile: Path,
+            outputFileName: Path, tasksDir: Path, font: String = PANDOC_DEFAULT_FONT) {
+    val commandLine = if (pandocCompileCommand == PANDOC_DEFAULT_VALUE) {
+        "$PANDOC_DEFAULT_VALUE $mainFile -o $outputFileName --pdf-engine xelatex -s -V mainfont='$font' "
```

Comment:
You may want to insert placeholders into the command line itself, like this:
"pandoc {0} -o {1} --pdf-engine xelatex..." and insert actual values with MessageFormat or Strings.format

Reply:
Done